### PR TITLE
New seo content

### PR DESCRIPTION
**Pull request by Daria-Voronina**
Please review a new file for Documentation and changes in the kr.tree file (a new toc-element).

### CODE REVIEW

File: docs/topics/cross-platform-mobile-development.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+Facebook introduced React Native in 2015 as an open-source framework designed to help mobile engineers build hybrid native/cross-platform apps. It’s based on ReactJS – a JavaScript library for building user interfaces. In other words, it uses JavaScript to build mobile apps for Android and iOS systems.
+
+React Native provides access to several third-party UI libraries with ready-to-use components, helping mobile engineers save time during the development process. Like Flutter, it allows you to see all your changes immediately, thanks to the Fast Refresh feature.
+
+You should consider using React Native for your app in the following cases:
+
+* Your application is relatively simple and is expected to be lightweight.
+* The development team is fluent in JavaScript or React.
+
+Applications built with React Native include Facebook, Instagram, Skype, Airbnb, and Uber Eats.
```
Comment (koshachy) [inline docs/topics/cross-platform-mobile-development.md:140]: AFAIK, Airbnb reverted back from React Native to native apps

File: docs/topics/cross-platform-mobile-development.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+Kotlin Multiplatform Mobile is an SDK for cross-platform mobile development provided by JetBrains. It allows you to create Android and iOS apps with shared logic. Its key benefits include:
+
+* Smooth integration with existing projects.
+* Full control over the UI, along with the ability to use the latest UI frameworks, such as SwiftUI and Jetpack Compose.
+* Easy access to Android and iOS SDKs without any restrictions.
+
+> Share the logic of your iOS and Android apps. Get started with [Kotlin Multiplatform Mobile](https://kotlinlang.org/docs/multiplatform-mobile-getting-started.html).
+>
+
+Global companies and start-ups alike have already leveraged Kotlin Multiplatform Mobile to optimize and accelerate their mobile development efforts. The benefits of this approach are apparent from the stories of the companies that have already adopted it.
```
Comment (koshachy) [inline docs/topics/cross-platform-mobile-development.md:153]:
```suggestion
> Share the logic of your iOS and Android apps. Get started with [Kotlin Multiplatform Mobile](https://kotlinlang.org/docs/multiplatform-mobile-getting-started.html).
>
{type="note"}

Global companies and start-ups alike have already leveraged Kotlin Multiplatform Mobile to optimize and accelerate their mobile development efforts. The benefits of this approach are apparent from the stories of the companies that have already adopted it.
```

File: docs/topics/cross-platform-mobile-development.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+However, many modern businesses need to reduce the time to market and the cost of per platform development while still aiming to have a presence both on Android and iOS. This is where cross-platform development frameworks like [Kotlin Multiplatform Mobile](https://kotlinlang.org/lp/mobile/) can help, as David Henry and Mel Yahya, a pair of senior software engineers from Netflix, [note](https://netflixtechblog.com/netflix-android-and-ios-studio-apps-kotlin-multiplatform-d6d4d8d25d23):
+
+> The high likelihood of unreliable network connectivity led us to lean into mobile solutions 
+> for robust client side persistence and offline support. The need for fast product delivery 
+> led us to experiment with a multiplatform architecture. Now we’re taking this one step further 
+> by using Kotlin Multiplatform to write platform agnostic business logic once in Kotlin 
+> and compiling to a Kotlin library for Android and a native Universal Framework for iOS.
+
+## Is cross-platform mobile development right for you?
```
Comment (koshachy) [inline docs/topics/cross-platform-mobile-development.md:61]:
```suggestion
> and compiling to a Kotlin library for Android and a native Universal Framework for iOS.
>
{type="tip"}

## Is cross-platform mobile development right for you?
```

**Review by koshachy — APPROVED** [review]
Overall is quite good.

Some minor comments, please take a look.
Also, there is a KMM abbreviation in the image, is it OK?

File: docs/topics/cross-platform-mobile-development.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
+[//]: # (title: What is cross-platform mobile development?)
+
+[//]: # (description: Cross-platform mobile development helps you save much time and effort. See why many developers have already switched to technology that is cost efficient.)
+
+Nowadays, many companies are facing the challenge of needing to build mobile apps for multiple platforms, specifically for both Android and iOS. This is why cross-platform mobile development solutions have emerged as one of the most popular software development trends.
+
+According to [Statista](https://www.statista.com/statistics/276623/number-of-apps-available-in-leading-app-stores/#:~:text=As%20of%20the%20first%20quarter,million%20available%20apps%20for%20iOS.), there were 3.48 million mobile apps available on the Google Play Store and 2.22 million apps on the App Store in the first quarter of 2021, with Android and iOS now accounting for [99%](https://gs.statcounter.com/os-market-share/mobile/worldwide) of the worldwide mobile operating system market.
```
[the remark this answers is not in the document]
Comment (Daria-Voronina) [reply docs/topics/cross-platform-mobile-development.md:7]: It’s strange, the link was accessible, but now I also see the “premium access” pop-up. Let’s delete the link then and keep the stats and the name of the source (Statista) only. WDYT?

[the remark this answers is not in the document]
Comment (Daria-Voronina) [reply docs/topics/cross-platform-mobile-development.md:7]: > Maybe we should make [99 of the worldwide mobile operating system market] as a link?
Agree!

File: docs/topics/cross-platform-mobile-development.md (hunk 1 above)
Comment (Daria-Voronina) [reply docs/topics/cross-platform-mobile-development.md:140]: I think you're right. I've Googled it and found a few recent resources that said that Airbnb stopped using React Native.

**Comment by Daria-Voronina** [comment]
Regarding the image with the KMM abbreviation, it's from one of the previous user surveys when this abbreviations was still used. I would keep it for some time if possible and then update it when designers will be available after finishing relocation. WDYT?

File: docs/topics/cross-platform-mobile-development.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/cross-platform-mobile-development.md:7]: Yes, it seems valid.
Will you change both links?

**Pull request merged**